Plots/Overlays/NCL_overlay_6.py

```diff
@@ -1,6 +1,6 @@
 """
 NCL_overlay_6.py
-================
+===============
```

Comment:
Sphinx will throw a warning with this saying that the title underline is too short. Make sure the underline is the same length as the title.